app/scripts/head/startup-styles.js

```diff
@@ -8,8 +8,6 @@
 
 // NOTE: This file must run in IE8+, so no ES5/ES6 features!
 
-//jshint es3: true, es5: false, esnext: false
```

Comment:
Is there any way to tell eslint that the files in `head` must all use es3 only syntax?